reactor-netty-core/src/main/java/reactor/netty/resources/ConnectionProvider.java

```diff
@@ -440,6 +442,10 @@ public ConnectionProvider build() {
 		static final Duration EVICT_IN_BACKGROUND_DISABLED       = Duration.ZERO;
 		static final int PENDING_ACQUIRE_MAX_COUNT_NOT_SPECIFIED = -2;
 
+		/**
+		 * Default timer service used for scheduling connection acquisition timers.
+		 */
+
```

Comment:
Is this javadoc needed?

Reply:
This is a mistake, I'm removing it, sorry.